setup.py

```diff
@@ -76,16 +75,46 @@ def write_version_file():
 pillow_req = 'pillow-simd' if get_dist('pillow-simd') is not None else 'pillow'
 requirements.append(pillow_req + pillow_ver)
 
+third_party_libraries = ['png', 'turbojpeg']
```

Comment:
Is there a reason you chose this over adding it to the [existing CMakeLists.txt](https://github.com/pytorch/vision/blob/master/CMakeLists.txt) file?